### DISCUSSION TITLE
Clearing / Replacing ComboboxInput value (Vue)

### DISCUSSION THREAD

**Discussion by louisjeck** [tailwindlabs/headlessui#1143] [Help]
Hello, 
Is there any way to override the ComboboxInput value ? It doesn't seem possible to use a v-model binding. I need to be able to set a default value (current value when editing a form) and clear the input, but I can't find how.

Edit: Seems I can mutate the :value prop to replace the value. Still I think the ComboboxInput should update its value whenever the value of the Combobox it mutated. As of now it doesn't seem to follow.

Thank you

**Comment by jameswong3388** [comment]
same problems here but not sure how to do it
